# config/named_poses.yaml
named_poses:
  home:
    position:
      x: 0.7
      y: -0.15
      z: 0.955
    orientation:
      x: 0.0
      y: 0.707
      z: 0.0
      w: 0.707
  
  ready:
    position:
      x: 0.3
      y: 0.0
      z: 0.3
    orientation:
      x: 0.0
      y: 0.0
      z: 0.0
      w: 1.0
  
  pick:
    position:
      x: 0.4
      y: 0.2
      z: 0.15
    orientation:
      x: 0.0
      y: 0.707
      z: 0.0
      w: 0.707
  
  place:
    position:
      x: 0.2
      y: -0.3
      z: 0.25
    orientation:
      x: 0.0
      y: 0.707
      z: 0.0
      w: 0.707
  
  inspect:
    position:
      x: 0.5
      y: 0.0
      z: 0.4
    orientation:
      x: 0.0
      y: 0.0
      z: 0.0
      w: 1.0